src/getImports/testVuln.js

```diff
@@ -63,6 +64,7 @@ function testWithAuth(pkg) {
 
 export default function test(pkg) {
```

Comment:
We should introduce some tests around `test` to validate that it is calling through to the statistics code